### ISSUE TITLE
In Xray all assertions should be added to steps in form of text

### ISSUE THREAD

**Issue by qytera-development** [Qytera-Gmbh/QTAF#209]
QTAF should create a comment for each test step execution that contains the information about the assertions that a test step checked. I would like to have a text like this:

```console
check table

PASSED ASSERTIONS
- Expected title cell's value to be "abc", the actual value was "abc"

FAILED ASSERTIONS
- Expected amount cell's value to be 25, the actual value was 50
```

**Comment by csvtuda** [comment]
Addressed in #212.

**Comment by OliverHoenig** [comment]
This feature has been worked on. It is now implemented in version 0.2.8.
